internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -795,10 +796,17 @@ func convertPublish(topics []manifest.Topic, accountID, region, app, env, svc st
 		return nil, err
 	}
 	var publishers template.PublishOpts
+	var topicType string
 	// convert the topics to template Topics
 	for _, topic := range topics {
+		if topic.Type != nil {
+			topicType = aws.StringValue(topic.Type)
+		} else {
+			topicType = defaultTopicType
+		}
```

Comment:
nit: Usually, in the presence of an `else` statement I try to find ways of getting rid of it as long as the operation is cheap:
```suggestion
		topicType := defaultTopicType
		if topic.Type != nil {
			topicType = aws.StringValue(topic.Type)
		}
```